src/math/nfft/dnfft_1d_kernels.cu

```diff
@@ -113,7 +115,7 @@ __global__ void sumKernel(const ScalarType* in, const int ldi, ScalarType* out,
   const int i = blockIdx.x * blockDim.x + threadIdx.x;
   const int j = blockIdx.y * blockDim.y + threadIdx.y;
 
-  out[i + ldo * j] += in[i + ldo * j];
+  out[i + ldo * j] += in[i + ldi * j];
```

Comment:
This sort of mistakes can be avoided by extending the matrix view class to device code, as it is done in the ct_int branch: https://github.com/CompFUSE/DCA-develop/blob/ct_int/include/dca/linalg/matrix_view.hpp.
Though I suppose we can do it after we merged the rest of gpu_trunk.